tests/test_gameboard.py

```diff
@@ -6,311 +6,355 @@
+        """
+        . . . . . . .
+        . . . . . . .
+        . . . . . . B
+        . . . . . . B
+        . W W W . . B
+        """
+        test_board.put_chip(1, 'W')
+        self.assertEqual(test_board.winner_exists(), True)
 
     def test_winner_exists_from_rows_on_right_corner(self):
-        self.assertEqual(self.board_test_rows.winner_exists(), False)
-        self.board_test_rows.put_chip(7, 'red')
-        self.assertEqual(self.board_test_rows.winner_exists(), True)
+        test_board = self.builder.build_from_moves([6,1,5,1,4,1])
+        """
+        . . . . . . .
+        . . . . . . .
+        B . . . . . .
+        B . . . . . .
+        B . . W W W .
+        """
+        test_board.put_chip(7, 'W')
+        self.assertEqual(test_board.winner_exists(), True)
 
     def test_winner_exists_from_columns_on_left_corner(self):
-        self.assertEqual(self.board_test_columns.winner_exists(), False)
-        self.board_test_columns.put_chip(1, 'blue')
-        self.assertEqual(self.board_test_columns.winner_exists(), True)
+        test_board = self.builder.build_from_moves([1,2,1,3,1,4])
+        """
+        . . . . . . .
+        . . . . . . .
+        W . . . . . .
+        W . . . . . .
+        W B B B . . .
+        """
+        test_board.put_chip(1, 'W')
+        self.assertEqual(test_board.winner_exists(), True)
 
     def test_winner_exists_from_columns_on_right_corner(self):
-        self.assertEqual(self.board_test_columns.winner_exists(), False)
-        self.board_test_columns.put_chip(7, 'blue')
-        self.assertEqual(self.board_test_columns.winner_exists(), True)
+        test_board = self.builder.build_from_moves([7,1,7,2,7,3])
+        """
+        . . . . . . .
+        . . . . . . .
+        . . . . . . W
+        . . . . . . W
+        B B B . . . W
+        """
+        test_board.put_chip(7, 'W')
+        self.assertEqual(test_board.winner_exists(), True)
 
     def test_winner_exists_from_diagonals_on_left_corner(self):
-        self.assertEqual(self.board_test_diagonals.winner_exists(), False)
-        self.board_test_diagonals.put_chip(1, 'red')
-        self.assertEqual(self.board_test_diagonals.winner_exists(), True)
+        test_board = self.builder.build_from_moves([1,2,1,2,1,2,2,1,6,3,3,6,3,3,7,4,4,5])
+        test_board.put_chip(1, 'W')
+        self.assertEqual(test_board.winner_exists(), True)
 
     def test_winner_exists_from_diagonals_on_right_corner(self):
-        self.assertEqual(self.board_test_diagonals.winner_exists(), False)
-        self.board_test_diagonals.put_chip(7, 'red')
-        self.assertEqual(self.board_test_diagonals.winner_exists(), True)
+        test_board = self.builder.build_from_moves([7,6,7,6,7,6,6,7,2,5,5,2,5,5,1,4,4,3])
+        """
+        . . . . . . .
+        . . . . B W B
+        . . . . W B W
+        . B . W W B W
+        W W B B B B W
+        """
+        test_board.put_chip(7, 'W')
```

Comment:
Just to make sure the winner didn't exist before, you should add an `assertFalse(test_board.winner_exists())` before the `put_chip`